src/main/resources/application.properties

```diff
@@ -1,2 +1,11 @@
 spring.application.name=dropwizard.to.springboot
 management.endpoints.web.exposure.include=health,info
```

Comment:
Check if it also shows database health check, if not then create one for Database using query `SELECT 1`, code will be similar to dropwizard one.